src/com/facebook/buck/jvm/kotlin/KotlinLibrary.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2016-present Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain
+ * a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.facebook.buck.jvm.kotlin;
+
+import com.facebook.buck.jvm.java.JavaLibrary;
+
+public interface KotlinLibrary extends JavaLibrary {
```

Comment:
I'm assuming you're going to add more stuff in here in later PRs? Can we either 1) hold off until we actually populate this with new stuff or 2) add a comment explaining why this is hear (and different from JavaLibrary)